### PR TITLE
Updated map API from 'kortforsyningen' to 'datafordeler'

### PR DESCRIPTION
**Pull request by 0xUnicorn**
Refactored existing MapProxyView class and made changes to endpoints+query in kfmap.js

Updated environment_settings.py with variables for credentials and removed the old token variable.

HOW TO:

Signup at https://datafordeler.dk

- Create a 'webbruger'
  - https://confluence.sdfe.dk/pages/viewpage.action?pageId=13665185
- Create a 'tjenestebruger'
  - https://confluence.sdfe.dk/x/0oPQ

Update 'DATAFORDELER_USER' & 'DATAFORDELER_PASSWORD' in settings.py (I've added ansible placeholders with same names)

### CODE REVIEW

File: src/maps/views.py [hunk 1]
```diff
@@ -52,3 +66,35 @@ def get(self, *args, **kwargs):
...
+            logger.warning("Maps endpoint was invalid: '%s' valid endpoints: %s",
+                           endpoint, self.VALID_ENDPOINTS)
+            raise PermissionDenied("No thanks")
+
+    def sanitize_path(self, path: str) -> str:
+        """Sanitize path by removing PROXY_URL and set 'transparent' value to upper
+        """
+        new_path = path.replace(self.PROXY_URL, "", 1)
+        sanitized_path = re.sub(
+            r"(transparent=)(true|false)",
```
Comment (tykling) [inline src/maps/views.py:84]: what does transparent do?

**Review by tykling — COMMENTED** [review]
looks very nice, thank you, just one comment (inline)

File: src/maps/views.py (hunk 1 above)
Comment (0xUnicorn) [reply src/maps/views.py:84]: To be honest.. No idea. But during my tests it was a required parameter for calling the API, and it wants the "true/false" value to be uppercase.

Comment (tykling) [reply src/maps/views.py:84]: :+1:

**Pull request merged**